rules/xcodeproj.bzl

```diff
@@ -64,21 +64,28 @@ def _xcodeproj_aspect_impl(target, ctx):
     bazel_build_target_name = ""
     if target.label.workspace_name != "":
         bazel_build_target_name = "@%s//" % target.label.workspace_name
-    bazel_build_target_name += "%s:%s" % (target.label.package, target.label.name)
```

Comment:
in the future, can unrelated formatting changes be made in a separate PR? it makes reviewing easier